index.page.tsx

```diff
@@ -487,27 +487,22 @@ function ContentItem(
   return (
     <div>
       <h4 className="text-lg font-semibold mb-1">{props.title}</h4>
-      <p className="mb-3">
-        {props.description}
-        {" "}
-      </p>
-      <a
-        className={`homepage-link ${productClass}`}
-        href={props.link}
-      >
-        {props.linktext} <span aria-hidden="true">-&gt;</span>
+      <p className="mb-3">{props.description}</p>
+      <a className={`homepage-link ${productClass}`} href={props.link}>
+        {props.linktext}{" "}
+        <span aria-hidden="true" class="whitespace-pre">
+          -&gt;
```

Comment:
why not use a unicode arrow character here? EG ➞

Reply:
The arrow ligature is actually part of the Inter typeface, so it goes with the text, wheres a unicode arrow will unavoidably look unicode-y. Here's an example (the right arrow is the unicode arrow):

![CleanShot 2024-08-09 at 13 20 02@2x](https://github.com/user-attachments/assets/54e7f914-0dfa-44a8-900d-2516e7726e65)